sequencer/src/api/options.rs

```diff
@@ -152,6 +162,30 @@ pub struct Submit;
 #[derive(Parser, Clone, Copy, Debug, Default)]
 pub struct Status;
 
+/// Options for the query API module backed by a Postgres database.
+#[derive(Parser, Clone, Debug, Default)]
+pub struct Sql {
+    /// Hostname for the remote Postgres database server.
+    #[clap(long, env = "ESPRESSO_SEQUENCER_POSTGRES_HOST")]
```

Comment:
For later, but I think we could use a URL instead ( https://docs.rs/tokio-postgres/latest/tokio_postgres/config/struct.Config.html#url ).

Reply:
Hmm I'm not sure I like that. I think having a separate setting for each parameter makes it easier to read and search for in e.g. .env files or terraform.

Maybe we could support both: ie the config is first parsed from a URL and then individual settings can be overridden via env vars

Reply:
I just thought since we usually go with URLs for clients. I agree postgres URLs can be a bit complex. I'm fine keeping what we have.